src/TraditionalMapFileParser.cc

```diff
@@ -361,7 +361,7 @@ namespace ChimeraTK::detail {
       channels.emplace_back(NumericAddressedRegisterInfo::ChannelInfo{uint32_t(channel.address - pl.address) * 8,
           channel.type, channel.width, channel.nFractionalBits, channel.signedFlag});
       bytesPerBlock += channel.nBytes;
-      if(channel.nBytes != 1 && channel.nBytes != 2 && channel.nBytes != 4) {
+      if(channel.nBytes != 1 && channel.nBytes != 2 && channel.nBytes != 4 && channel.nBytes != 8) {
```

Comment:
Just a comment. This is the proper extension for the check for primitive types, so the change is fine like this. However, I don't know if this is a necessary restriction. Why can the alignment not be  3 bytes?